app/decorators/user_decorator.rb

```diff
@@ -21,16 +21,12 @@ class UserDecorator < ApplicationDecorator
   DEFAULT_PROFILE_SUMMARY = "404 bio not found".freeze
 
   def cached_followed_tags
-    follows_map = Rails.cache.fetch("user-#{id}-#{following_tags_count}-#{last_followed_at&.rfc3339}/followed_tags",
-                                    expires_in: 20.hours) do
-      Follow.follower_tag(id).pluck(:followable_id, :points).to_h
+    Rails.cache.fetch(
+      "user-#{id}-#{following_tags_count}-#{last_followed_at&.rfc3339}/followed_tags",
```

Comment:
this key's name isshared between the user and the user decorator
https://github.com/forem/forem/blob/main/app/models/user.rb#L321-L332

Reply:
@djuber It's very close but it's not quite the same. The one in the user model ends in `followed_tag_names`. Technically safe, but it's definitely easy to confuse them, so I'm not sure if we should make them more distinct.